src/main/java/io/codelex/java_advanced_test/Exercise3/Joiner.java

```diff
@@ -0,0 +1,30 @@
+package io.codelex.java_advanced_test.Exercise3;
+
+public class Joiner<T> {
+
+    private final T value;
+
+    public Joiner(T value) {
+        this.value = value;
+    }
+
+    public String join(T value) {
```

Comment:
Ok, redzu, ka šis strādā, tikai doma bija nedaudz cita. Uzdevumā rakstīts:
"join() method that allows you to specify any number of T-type objects" Tā tad jāspēj ir padot dažādu skaitu ar objektiem - šobrīd šī metode var saņemt tikai 1 objektu. Būtu jāpārtaisa par varargs, jebšu T... values